server/src/db/model.ts

```diff
@@ -91,6 +98,50 @@ const updateUser = (
         },
         { upsert: true }
     );
+}
+
+function updateExercises(
+    id,
+    interests: String,
+    fitnessLevel: String,
+    name: String,
+    sets: Number,
+    reps: Number
+) {
+    const updatedUser = User.findByIdAndUpdate(
```

Comment:
This seems incorrect, since it will change the entire exercise field to one exercise object